safe_extras/parameters/string_parameter.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, guid=None):
         :param guid: Optional unique identifier for this parameter. If none
             is specified one will be generated using python hash. This guid
             will be used when storing parameters in the registry.
-        :type guid: str
+        :type guid: str, None
         """
         super(StringParameter, self).__init__(guid)
-        self.expected_type = [str, unicode]
```

Comment:
Hi @cchristelis thanks for pointing this out. I think there was someone adding unicode directly to safe_extras/parameters. That's why it will have some different with the original repo.

I will update the original repo (https://github.com/AIFDR/parameters), and fix this.

Thanks